Gems/ROS2/Code/Include/ROS2/Manipulator/JointPublisherComponent.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <ROS2/Manipulator/MotorizedJointComponent.h>
+#include <RobotImporter/URDFMetadataComponent.h>
+#include <rclcpp/publisher.hpp>
+#include <sensor_msgs/msg/joint_state.hpp>
+#include <sensor_msgs/msg/joint_state.hpp>
+#include <control_msgs/action/follow_joint_trajectory.h>
+#include <AzCore/Component/TickBus.h>
+
+namespace ROS2
+{
+    //! A component responsible for storing the jointComponent tree structure
+    //! and each of the joint's name as they are described in URDF
```

Comment:
The comment could describe in some more detail what the jointComponent tree structure is and what `each of the joint's name` is/are?